### DISCUSSION TITLE
Data viewer wider columns for text data

### DISCUSSION THREAD

**Discussion by StefanMunnes** [posit-dev/positron#4998] [Feedback]
Hello and thank you all for your great work, I'm heavily using and enjoying Positron since a few weeks. One restriction I encounter quite often is the very limited width of the columns in the data viewer. This is mainly problematic, working a lot with textual data, because when inspecting and searching the data I'm limited to just the first few words. In VSCodes native data viewer, I was able to expand the column width much more and in RStudio there appears the whole text in a hovering box when inspecting a cell with the mouse.
You think it's possible to implement such features?

**Comment by juliasilge** [comment]
I believe this is the same as https://github.com/posit-dev/positron/issues/3658. Want to take a look and see if you have additional detail to add?

**Reply by StefanMunnes** [reply]
Yes, plus manual expansion of the column width with the mouse. At the moment it's almost impossible for me to see e.g. longer URLs in full because I can only expand the column up to a certain width. Hovering and tooltip boxes showing the content are nice, but seeing all the content (e.g. longer URLs) across multiple rows at once would be even better.
